test/python/sdpa/frost/test_sdpa_fwd_split_kv_sm100.py

```
@@ -1135,7 +1148,7 @@ def qz(shape):
             extra = dict(sf_q=sf_q, sf_k=sf_k, sf_v=sf_v)
             kw = {}
         else:
-            mk = lambda *sh: (torch.randn(*sh, device=dev) * 0.5).clamp(-448, 448).to(torch.float8_e4m3fn)
+            mk = lambda *sh: (torch.randn(*sh, device=dev) * 0.5).to(fp8_dtype)
```

Ruff (0.16.3)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `mk` as a `def`

(E731)